packages/error-reporting/src/google-apis/auth-client.js

```diff
@@ -97,6 +97,17 @@ class RequestHandler extends common.Service {
     }, config);
     this._config = config;
     this._logger = logger;
+
+    var that = this;
+    this.authClient.getToken(function(err, token) {
+      if (err) {
+        that._logger.warn([
+          'Unable to find credential information on instance. This library ' +
+          'will be unable to communicate with the Stackdriver API to save ' +
+          'errors.  Message: ' + err.message
```

Comment:
I think you should either use `+` or `join(' ')` to concatenate these strings, but not both.
Additional nit: leave single space after period on this line.